stories/Megamenu/MegamenuExample.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+
+import { Megamenu, LinkList, LinkListItem, Navbar, Icon } from '../../src'
+
+class MegamenuExample extends React.Component {
```

Comment:
I have made this a function component but the megamenu only toggles (open to close or vice versa) only once. I think this is because reactstrap Dropdown needs a `toggle` prop as well which is a function

Reply:
I have replaced `Dropdown` with `UncontrolledDropdown` so this is not a problem anymore